### PR TITLE
specviz2d: cli support (2d spectrum only)

### PR DESCRIPTION
**Pull request by kecnry**
### Description

This pull request adds basic (2d spectrum only) CLI support for specviz2d.  Support for passing a 1D spectrum filename may require refactoring (either a more advanced parser or an additional CLI argument only used for specviz2d) [:cat:](https://jira.stsci.edu/browse/JDAT-2686) - filed as #1578 for future work.

Needed for upcoming specviz2d docs PR (docs not included as part of this PR since the docs for all of specviz need to be populated).

As far as I know, we don't have any way to include the CLI in the CI tests, so 0% coverage is expected.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [x] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [x] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Did the CI pass? If not, are the failures related?
- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`?
- [x] Is a milestone set?
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: jdaviz/configs/specviz2d/specviz2d.ipynb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# PREFIX\n",
+    "from jdaviz import Specviz2d\n",
+    "\n",
+    "specviz = Specviz2d(verbosity='JDAVIZ_VERBOSITY', history_verbosity='JDAVIZ_HISTORY_VERBOSITY')\n",
+    "specviz.load_data('DATA_FILENAME')\n",
+    "specviz.app"
```
Comment (pllim) [inline jdaviz/configs/specviz2d/specviz2d.ipynb:14]: Did we forget to update this in the CLI notebooks?

```suggestion
    "specviz.show()"
```

File: jdaviz/configs/specviz2d/specviz2d.ipynb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.6-final"
```
Comment (pllim) [inline jdaviz/configs/specviz2d/specviz2d.ipynb:34]: p.s. Tsk tsk, you still using Python 3.7? Astropy doesn't support it anymore, so you are not getting the latest astropy in that stack.

File: CHANGES.rst [hunk 3]
```diff
@@ -42,6 +42,8 @@ Specviz2d
 
 - Spectral extraction plugin. [#1514, #1560, #1562]
 
+- CLI support for launching Specviz2d (2D spectrum only). [#1576]
```
Comment (pllim) [inline CHANGES.rst:45]: I don't understand the emphasize on "2D spectrum only" if I were a user and reading this entry without further insider knowledge. Please explain more.

**Comment by kecnry** [comment]
I just duplicated the existing template notebook for specviz and swapped out for specviz2d.  If we want to make your suggested changes, I think we should make them across all of the template notebooks.  Does that belong here or as a separate PR (it's easy enough to do either way)?

File: CHANGES.rst (hunk 3 above)
Comment (kecnry) [reply CHANGES.rst:45]: The CLI which is shared across all configs only accepts a single path (filename or directory).  The specviz2d parser currently takes either 1 filename (2d spectrum, automatically extracting the 1d spectrum) or 2 filenames (2d spectrum and 1d spectrum).  Supporting that will either take significant work to the parser, or an additional argument in the CLI just for specviz2d - as explained in the PR description.  I think that decision will require discussion but shouldn't necessarily block the basic support introduced here.  I'm open to suggestions for making that more clear in the changelog though without getting too technical!

**Comment by pllim** [comment]
> Does that belong here or as a separate PR

Seems so trivial that I think you should just update them all here if you want. If not, separate PR is ok too. As long as changing from `.app` to `.show()` doesn't break CLI somehow... pinging @duytnguyendtn .

File: CHANGES.rst (hunk 3 above)
Comment (pllim) [reply CHANGES.rst:45]:
```suggestion
- CLI support for launching Specviz2d for a single 2D spectrum file input.
  Use notebook version if you want to open separate 2D and 1D spectra in Specviz2d. [#1576]
```

**Comment by kecnry** [comment]
arguably `.app` might be slightly cheaper, so for notebooks that the users shouldn't ever see, we might want to leave it as is to avoid the logic in `.show()`?

**Comment by pllim** [comment]
Okay, let's leave `.app` alone then. Thanks for the clarification. I'll approve after you resolve the conflict.

**Review by pllim — APPROVED** [review]
LGTM. Thanks!

**Review by ojustino — APPROVED** [review]
This worked for me in Firefox and Safari.

**Comment by kecnry** [comment]
Thanks for the reviews!

**Pull request merged**